core/api/v1/keytransparency.proto

```diff
@@ -474,8 +484,15 @@ service KeyTransparency {
   // update appears after a timeout.
   rpc QueueEntryUpdate(UpdateEntryRequest) returns (google.protobuf.Empty) {
     option (google.api.http) = {
-      post: "/v1/directories/{directory_id}/users/{user_id}:queue"
+      post: "/v1/directories/{directory_id}/users/{entry_update.user_id}:queue"
       body: "entry_update"
     };
   }
+  // BatchQueueUser enqueues a list of user profiles.
+  rpc BatchQueueUser(BatchQueueUserRequest) returns (google.protobuf.Empty) {
```

Comment:
The naming seems weird - you're queuing an update for a user, not queuing a user. What about `BatchQueueUserUpdate`, with the custom verb `:batchQueueUpdate`?

That'd fall slightly afoul of the Cloud pattern of having API endpoints named VerbResource, but `BatchQueueUpdateUser` also strikes me as odd.

Reply:
That sounds good.  
I think the reason `BatchQueueUpdateUser` sounds odd is because `Queue` and `Update` are both verbs :-)